app/views/layouts/_nav.html.erb

```diff
@@ -3,18 +3,18 @@
     <%= link_to t(".dovetail"), main_app.podcasts_path, class: "navbar-brand" %>
 
     <ul class="navbar-nav">
-      <li class="nav-item">
+      <li class="nav-item d-none d-md-block">
```

Comment:
I decided to hide these links on mobile devices. I suspect if you are accessing Feeder on your phone, you will only need to navigate feeder.

NOTE: When we get to implementing this on ID, we may have to do some more work to get these links to play nicer in mobile (but ID won't have a subnav)